cypress/e2e/main-page.cy.js

```diff
@@ -26,36 +19,38 @@ describe('Main page', function () {
     cy.contains('PyPI - License')
```

Comment:
TODO: This test should pass in CI but fails locally because search only works with a compiled frontend. Think I need to bin off the ability to run cypress against a dev build